src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -1208,16 +1208,21 @@ class TypeSpecTest {
         | * A hard or soft tortilla, loosely folded and filled with whatever
         | * [random][java.util.Random] tex-mex stuff we could find in the pantry
         | * and some [kotlin.String] cheese.
-        | *
-        | * @param temperature Taco temperature. Can be as cold as the famous ice tacos from
-        | * the Andes, or hot with lava-like cheese from the depths of
-        | * the Ninth Circle.
-        | * @property soft True for a soft flour tortilla; false for a crunchy corn tortilla.
-        | * @property mild No one likes mild tacos.
         | */
         |class Taco(
+        |    /**
+        |     * Taco temperature. Can be as cold as the famous ice tacos from
+        |     * the Andes, or hot with lava-like cheese from the depths of
+        |     * the Ninth Circle.
+        |     */
         |    temperature: Double,
+        |    /**
+        |     * True for a soft flour tortilla; false for a crunchy corn tortilla.
+        |     */
         |    val soft: Boolean,
+        |    /**
+        |     * No one likes mild tacos.
```

Comment:
hmm, would it be better to override the parameter kdoc with any property kdoc, possibly throwing an exception to alert the author of this potential conflict?

Reply:
Decided to go with putting parameter KDoc under the type/primary constructor header if there's a conflict.